maintenance/Rector/MockGrabyResponseRector.php

```diff
@@ -42,21 +42,9 @@ final class MockGrabyResponseRector extends AbstractRector
     private const FIXTURE_DIRECTORY = __DIR__ . '/../../tests/fixtures/content';
     private const MATCHING_ERROR_COMMENT = 'TODO: Rector was unable to evaluate this Graby config.';
     private const IGNORE_COMMENT = 'Rector: do not add mock client';
-    private BetterNodeFinder $betterNodeFinder;
```

Comment:
I assume these changes are made by rector but have no bearing on ability to run the code. It would be easier to review if they were split into a separate commit.

Reply:
Yeah it was done by rector as `maintenance` folder is part of the config.